pkg/testing/deepcopy_test.go

```diff
@@ -74,6 +76,31 @@ func TestRoundTripTypesWithoutProtobuf(t *testing.T) {
 	}
 }
 
+func TestFailRoundTrip(t *testing.T) {
+	scheme := runtime.NewScheme()
+	codecs := serializer.NewCodecFactory(scheme)
+	groupVersion := schema.GroupVersion{Group: "broken", Version: "v1"}
+	builder := runtime.NewSchemeBuilder(func(scheme *runtime.Scheme) error {
+		scheme.AddKnownTypes(groupVersion, &BrokenType{})
+		metav1.AddToGroupVersion(scheme, groupVersion)
+		return nil
+	})
+	builder.AddToScheme(scheme)
+	seed := rand.Int63()
+	fuzzer := fuzzer.FuzzerFor(genericfuzzer.Funcs, rand.NewSource(seed), codecs)
+	gvk := groupVersion.WithKind("BrokenType")
+	roundtrip.RoundTripSpecificKindWithoutProtobuf(t, gvk, scheme, codecs, fuzzer, nil)
+	// It's very hacky way of making sure the DeepCopy is actually invoked inside RoundTripSpecificKindWithoutProtobuf
+	// used in the other test. If for some reason this tests starts passing we need to fail b/c we're not testing
+	// the DeepCopy in the other method which we care so much about.
+	if t.Failed() {
+		os.Exit(0)
```

Comment:
doesn't this skip the other test coming after this?

Reply:
Fixed with a temporary `testing.T`